jwst/regtest/test_miri_image.py

```diff
@@ -55,9 +55,10 @@ def run_pipelines(jail, rtdata_module):
     # image3 pipeline on all _cal files listed in association
     rtdata.get_data("miri/image/det_dithered_5stars_image3_asn.json")
     args = ["config/calwebb_image3.cfg", rtdata.input,
-        # Set some unique tweakreg param values needed for these data
+        # Set some unique param values needed for this data
```

Comment:
I beg to differ: "data" is plural (as opposed to, say, a data file), hence "these" instead of "this".

<#grammar-police>

Reply:
Not sure what your agendum is here Howard, but I will change it back.  😁